app/assets/stylesheets/application.scss

```diff
@@ -24,8 +20,16 @@ body {
 }
 
 .master-overlay {
-  background-color: rgba(250, 250, 250, 0.2);
+  background-color: rgba(41, 88, 220, 0.41);
   padding: 2vw;
+  border-radius: 50% 50%;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `50%` instead of `50% 50%`